toppra/algorithm/reachabilitybased/reachability_algorithm.py

```diff
@@ -338,3 +338,51 @@ def compute_parameterization(self, sd_start, sd_end, return_data=False):
             return sdd_vec, sd_vec, v_vec, K
         else:
             return sdd_vec, sd_vec, v_vec
+
+
+    def _one_step_forward(self, i, L_current, feasible_set_next):
```

Comment:
This function's name should be `_reach` to maintain consistency with the paper. :+1: